style-plugin.js

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+const postcss = require('postcss');
+const postCssModules = require('postcss-modules');
+const sass = require('sass');
+
+const CreateStylePlugin = isProd => {
```

Comment:
it might be helpful to include some links if there are any references you started from while designing this